app/models/reservation.rb

```diff
@@ -141,6 +141,7 @@ def approved?
   def flagged?(flag)
     # checks to see if the given flag is set
     # you must pass the symbol for the flag
+    return unless FLAGS.key? flag
```

Comment:
So we're returning `nil` instead of  `false` if they pass a key that we don't have? The guard clause makes sense, but what do you think about returning `false` instead?